testing/bchunk/APKBUILD

```diff
@@ -0,0 +1,33 @@
+# Contributor: Alexander Edland <alpine@ocv.me>
+# Maintainer: Alexander Edland <alpine@ocv.me>
+pkgname=bchunk
+pkgver=1.2.2
+pkgrel=0
+pkgdesc="Convert bin+cue CD images to .iso and .cdr"
+url="https://github.com/hessu/bchunk"
+arch="all"
+license="GPL-2.0-or-later"
+options="!check"  # no test suite
+subpackages="$pkgname-doc"
+source="bchunk-$pkgver.tar.gz::https://github.com/hessu/bchunk/archive/release/$pkgver.tar.gz"
+builddir="$srcdir/bchunk-release-$pkgver"
+
+build() {
+	cd "$builddir"
+	make
+}
+
+package() {
+	cd "$builddir"
+	mkdir -p \
+		"$pkgdir/usr/bin" \
+		"$pkgdir/usr/share/man/man1"
+	
+	sed -ri '/^(BIN|MAN)_DIR /d' Makefile
```

Comment:
This shouldn't be needed, as `make X=Y` on the command line will override variable `X` in the Makefile already.

Reply:
you're right - must have been a leftover I missed :p